.github/workflows/website.yml

```diff
@@ -33,25 +33,39 @@ jobs:
         run: |
           pip install -r requirements.txt
 
-      # Action to download artifacts from a different workflow (analysis.yml)
+      # Action Figures artifact
       - name: 'Download artifact'
         if: ${{ github.event.workflow_run.conclusion == 'success' }}
         uses: ./.github/actions/download-artifact
         with:
           name: 'Figures'
+      # Action analysis data artifact
+      - name: 'Download analysis data'
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: ./.github/actions/download-artifact
+        with:
+          name: 'Data'
 
-      - name: Build html
+      - name: move data to the website folder
+        run: |
+          mv test_output.csv website/
```

Comment:
I think we should probably avoid mixing build files with repo files. It looks like previously we were moving everything into a few folder and pushing from there. That's probably a slightly safer approach. Right now, for example, there's `combined_report.html` in the website folder. How does that interact with these new files?

Reply:
I created a separate folder `dashboard_website` where only the code needed for building the dashboard resides.